ipod/merge.py

```diff
@@ -38,6 +38,7 @@ def merge_and_extend_orbits(
     max_iter: int = 10,
     min_mjd: Optional[float] = None,
     max_mjd: Optional[float] = None,
+    astrometric_errors: dict[str, Tuple[float, float]] = ASTROMETRIC_ERRORS,
```

Comment:
Small risk of default values being dictionaries in function definitions. Should the dictionary get mutated, it will carry over to subsequent calls.